packages/shared/src/components/Sidebar.spec.tsx

```diff
@@ -31,7 +31,7 @@ const renderComponent = (
   alertsData: AlertContextData = {
     alerts: { filter: true },
     setAlerts: jest.fn(),
-    disableFilterAlert: disableAlertFilterMock,
+    updateAlerts: disableAlertFilterMock,
```

Comment:
should we not also change the mock name?

Reply:
Correct, we should!